packages/router/test/EggRouter.test.ts

```
@@ -0,0 +1,458 @@
+  });
+
+  it('should throw error on generator function', () => {
+    const app = {
+      controller: {
+        async foo() {
+          return;
+        },
+        hello: {
+          *world() {
+            return;
+          },
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)